### DISCUSSION TITLE
Merge successful but the pull shark not updated

### DISCUSSION THREAD

**Discussion by KorsChen** [community/community#75489] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

https://github.com/ShinChven/apple-storage-watcher/pull/3

My pull request was merged, but the number of pull requests in Pull Shark was not updated.

**Comment by mayachee** [comment]
Check Pull Shark Documentation or Verify Pull Request Status

**Comment by elderguardian** [comment]
Hey, this is the table for Pull Shark:

| Rank | PR needed |
|--- |---|
| x1 | over 2 PR |
| x2 | over 16 PR |
| x3 | over 128 PR |
| x4  |over 1024 PR |

From your profile, I can see that you are currently on the first stage.
I checked using the API and you currently only have 3 merged PRs.
You will need 13 more for the badge to get any updates.

You can view all your PRs at https://github.com/pulls

**Comment by qil50** [comment]
Interval Mode:

Configure a time interval for information requests.
Set the interval using "Request Interval."
ScheduleJob Mode:

Configure periodic scheduled runs for information requests.
Set the schedule using "Request Schedule," where you can use cron syntax for periodic execution.
Additional Information:

You need to input store codes and product codes to get started.
You can use cron syntax to define the timing for periodic runs in ScheduleJob.




**Pull Shark Badge System:**

| Rank | PR Needed      |
|------|----------------|
| x1   | Over 2 PR      |
| x2   | Over 16 PR     |
| x3   | Over 128 PR    |
| x4   | Over 1024 PR   |


**Badge Explanation:**

- **x1 Badge:**
  - **Requirement:** Achieve over 2 merged pull requests.
  - **Your Status:** Currently, you are on the first stage.
  - **Progress:** You currently have 3 merged pull requests.

- **x2 Badge:**
  - **Requirement:** Achieve over 16 merged pull requests.

- **x3 Badge:**
  - **Requirement:** Achieve over 128 merged pull requests.

- **x4 Badge:**
  - **Requirement:** Achieve over 1024 merged pull requests.

**Progress Update:**
- **Your Status:** From your profile, it's observed that you currently have 3 merged pull requests.
- **Next Step:** You will need 13 more merged pull requests to qualify for the x2 badge and receive any updates.

**View Your Pull Requests:**
You can view all your pull requests on [GitHub Pulls](https://github.com/pulls).

This system outlines the different badge ranks based on the number of merged pull requests, providing a clear progression path for contributors in the Pull Shark project. The explanation also gives a personalized update on the user's current status and the next steps needed to achieve the next badge.
